crates/turborepo-lib/src/daemon/client.rs

```diff
@@ -127,7 +127,7 @@ impl<T> DaemonClient<T> {
     pub async fn discover_packages(&mut self) -> Result<DiscoverPackagesResponse, DaemonError> {
         let req = proto::DiscoverPackagesRequest {};
         let mut req = req.into_request();
-        req.set_timeout(Duration::from_millis(10));
+        req.set_timeout(Duration::from_millis(50));
```

Comment:
This feels too high to me. With a debug build doing a local package discovery on the `turbo` repo takes 30ms.

Reply:
Changing to 30. Will adjust again in another PR after some additional fixes